apps/quickbooks_online/utils.py

```
@@ -1075,6 +1065,13 @@ def __construct_credit_card_purchase(self, credit_card_purchase: CreditCardPurch
 
         credit_card_purchase_payload = self.purchase_object_payload(credit_card_purchase, line, account_ref=credit_card_purchase.ccc_account_id, payment_type='CreditCard', doc_number=credit_card_purchase.credit_card_purchase_number, credit=credit)
 
+        if general_settings.is_multi_currency_allowed and fyle_home_currency != qbo_home_currency and qbo_home_currency:
+            exchange_rate = self.connection.exchange_rates.get_by_source(source_currency_code=fyle_home_currency)
+            credit_card_purchase_payload['ExchangeRate'] = exchange_rate['Rate'] if "Rate" in exchange_rate else 1
```

Ruff (0.8.2)

Use `exchange_rate.get('Rate', 1)` instead of an `if` block

Replace with `exchange_rate.get('Rate', 1)`

(SIM401)